devices.js

```diff
@@ -5961,6 +5961,37 @@ const devices = [
             await configureReporting.batteryPercentageRemaining(endpoint);
         },
     },
+    {
+        zigbeeModel: ['SZ-DWS04N_SF'],
+        model: 'SZ-DWS04N_SF',
+        vendor: 'Sercomm',
+        description: 'Magnetic door & window contact sensor',
+        supports: 'contact',
+        fromZigbee: [fz.ias_contact_alarm_1, fz.temperature, fz.battery_3V_2100],
+        toZigbee: [],
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement', 'genPowerCfg']);
+            await configureReporting.temperature(endpoint);
+            await configureReporting.batteryPercentageRemaining(endpoint);
+        },
+    },
+    {
+        zigbeeModel: ['SZ-PIR02_SF'],
+        model: 'SZ-PIR02_SF',
```

Comment:
Is this the device: https://www.amazon.de/Almond-Bewegungsmelder-Benachrichtungen-kostenlosen-erhalten/dp/B0172JJ6HY ?

Reply:
Yes, it is sold by Securifi, but self-reports as Sercomm